funnel/forms/project.py

```diff
@@ -110,6 +110,17 @@ class ProjectForm(forms.Form):
         validators=[forms.validators.DataRequired()],
         description=__("Landing page contents"),
     )
+    allow_rsvp = forms.BooleanField(__("Allow site visitors to RSVP (login required)"))
```

Comment:
These two fields should not be re-added to the form. Setting these fields is backend-only, or should be a special form that is site editor-only.